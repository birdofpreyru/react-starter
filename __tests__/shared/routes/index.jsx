import { JU } from '@dr.pogodin/react-utils';

import Routes from 'routes';

test('Matches shallow snapshot', () => {
  JU.shallowSnapshot(<Routes />);
});
