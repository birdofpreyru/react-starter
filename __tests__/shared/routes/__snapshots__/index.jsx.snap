// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Matches shallow snapshot 1`] = `
<Routes>
  <Route
    element={<Examples />}
    path="*"
  />
</Routes>
`;
